tools/scripts-config-react-webpack/config/webpack.config.js

```diff
@@ -201,6 +201,7 @@ async function getIndexTemplate(env, mode, indexTemplatePath, useInitiator = tru
 		headScript = `${renderMeta()}<base href="${BASENAME}" />
 		<script type="text/javascript">
 			window.basename = '${BASENAME}';
+			var process = { browser: true, env: { NODE_ENV: '${mode}' } };
```

Comment:
may be we can bring this out of "if else" WDYT ?
https://github.com/Talend/ui/pull/5060/files#diff-8747f8a488e054116fad4e5ca4a49eab72b66d12ea5641bbd2f340cb9da87607R189

Reply:
Not sure of the impact if we change the order of things in the Initiator part